sample-apps/react/react-dogfood/style/Whiteboard/Whiteboard.scss

```
@@ -0,0 +1,223 @@
+  flex: 1 1 auto;
+  min-height: 0;
+  overflow: hidden;
+  border-radius: var(--str-video__border-radius-md);
+}
+
+.rd__whiteboard__canvas {
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-color: $wb-surface;
+  // let pointer events drive drawing instead of browser scroll/zoom gestures
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)